app/views/newflow/login_signup/login_form.html.erb

```diff
@@ -71,7 +71,8 @@
                     <div class="content">
                         <%= f.submit I18n.t(:"login_signup_form.continue_button"),
                                     disable_with: I18n.t(:"login_signup_form.continue_button"),
-                                    class: 'primary'
+                                    class: 'primary', 
+                                    :data => {:ga_category => 'Login & Account Creation', ga_action: 'Continue', ga_label: 'Continue '}
```

Comment:
It would be really nice if we used the new Ruby syntax `key: 'value'` vs `:key => 'value'` everywhere.

Reply:
I tried this, but it threw an error when using key:value on the first item in the hash. `syntax error, unexpected ':', expecting =>`

Reply:
Something's not right. Try this has got to work:
`data: { ga_category : 'Login & Account Creation', ga_action: 'Continue', ga_label: 'Continue ' }`

Reply:
I made the change, pushed a commit.